utils/RepoUtils.js

```diff
@@ -183,4 +183,9 @@ module.exports = class RepoUtils {
     });
     return Array.from(new Set(r));
   }
+
+  static getOrganization(url) {
+    const regex = /datavisyn|phovea|caleydo|oltionchampari/;
```

Comment:
We should also think there more globally so that we don't have the organisations in the generator --> normally you could extract it from the url